### ISSUE TITLE
We anticipated loading the native portion of java-gnome from: /usr/lib/libgtkjni-4.1.3.so

### ISSUE THREAD

**Issue by logix2** [nikitabobko/gcal-notifier-kotlin-gtk#1]
It looks like gcal-notifier-kotlin-gtk tries to load the ```libgtkjni-4.1.3.so``` file from ```/usr/lib/``` and fails in Ubuntu 18.04. libgtkjni-4.1.3.so is in ```/usr/lib/jni/``` on my Ubuntu 18.04 system. 

Output:
```
$ gcal-notifier-kotlin-gtk
Exception in thread "main" java.lang.UnsatisfiedLinkError: 

We anticipated loading the native portion of java-gnome from:
/usr/lib/libgtkjni-4.1.3.so
but didn't find the library there.

	at org.freedesktop.bindings.Plumbing.loadNativeCode(Plumbing.java:205)
	at org.freedesktop.bindings.Plumbing.<clinit>(Plumbing.java:110)
	at org.gnome.gtk.Gtk.init(Gtk.java:95)
	at ru.nikitabobko.gcalnotifier.MainKt.main(Main.kt:12)
```

**Comment by nikitabobko** [comment]
Thank you for your feedback! I'll release the fix as soon as possible. As temp workaround you can do
`$ sudo ln -s /usr/lib/jni/libgtkjni-4.1.3.so /usr/lib/libgtkjni-4.1.3.so`

**Comment by nikitabobko** [comment]
Fixed on [v1.0.2](https://github.com/nikitabobko/gcal-notifier-kotlin-gtk/releases/tag/v1.0.2)

**Comment by nikitabobko** [comment]
Do not forget to remove symlink if you used my temp workaround suggestion:
`$ sudo rm /usr/lib/libgtkjni-4.1.3.so`
